PhysicsTools/Heppy/interface/JetUtils.h

```diff
@@ -0,0 +1,18 @@
+#ifndef PhysicsTools_Heppy_JetUtils_h
+#define PhysicsTools_Heppy_JetUtils_h
+
+#include "DataFormats/Common/interface/Ptr.h"
+#include "DataFormats/PatCandidates/interface/Jet.h"
+
+namespace heppy{
+
+  struct JetUtils{
+
+    static const pat::Jet 
+    copyJet(const edm::ProductID& id, const pat::Jet& ijet, unsigned long key);
```

Comment:
I'd suggest to have this just be `copyJet(const pat::Jet &jet)` and hide also the temporary `edm::Ptr` inside the function.
It should be both cleaner to read and faster.